src/tokens/solana.tokenlist.json

```diff
@@ -8527,6 +8527,23 @@
         "website": "https://www.solanasail.com",
         "twitter": "https://twitter.com/solsailsummer"
       }
+     },
+     {
+       "chainId": 101,
+       "address": "5GG1LbgY4EEvPR51YQPNr65QKcZemrHWPooTqC5gRPBA",
```

Comment:
this was needed to be replaced with at: https://github.com/solana-labs/token-list/pull/939